msteams-platform/overview-explore.md

```diff
@@ -31,6 +31,8 @@ With Teams, you can build your app in a feature-rich environment. Using Teams as
 
 Choose to customize your app with features and tools, such as user authentication, Microsoft Graph, and Developer Portal.
 
+Think you Teams personal tab app adds value to your user base in other Microsoft 365 platforms ? [Consider upgrading it run across the M365 ecosystem](/msteams-platform/m365-apps/overview.md).
+
```

Comment:
I notice the articles in this Overview section intentionally don't have lateral links into the content, in order to guide the reader through the flow to "The Teams solution" article (which has a collection of links across Teams content, including multi-hub). In that spirit, let's omit this text addition.

Reply:
remove the whole verbiage or just the link?

Reply:
The whole verbiage for now--